src/pages/Dashboard/SinglePackage.tsx

```
@@ -8,7 +8,7 @@ const SinglePackage = () => {
   const params = useParams();
   const navigate = useNavigate()
   const {data,isLoading} = useGetSinglePackageQuery(params.id) as any;
-  const {data:walletBalance, refetch} = useGetAccountBalanceQuery({}) as any;
+  const {data:walletBalance} = useGetAccountBalanceQuery({}) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)